youcubeapi.lua

```diff
@@ -0,0 +1,14 @@
+-- https://github.com/Commandcracker/YouCube
+
+local package = {}
+package.name = "youcubeapi"
+package.instdat = {}
+package.instdat.repo_owner = "commandcracker"
+package.instdat.repo_name = "youcube"
```

Comment:
For accuracy, these should be capitalized as they are in the repo:
```suggestion
package.instdat.repo_name = "YouCube"
```